app/helpers/image_encode_helper.rb

```diff
@@ -0,0 +1,33 @@
+module ImageEncodeHelper
+  def encodable_image_tag(source, options = {})
+    image_tag(encodable_image_source(source), options)
+  end
+  alias eimage_tag encodable_image_tag
+
+  def encodable_image_source(source)
+    if base64_encode_images? && source.present?
+      base64_encoded_uri(source)
+    else
+      path_to_image(source)
+    end
+  end
+  alias eimage_source encodable_image_source
+
+  def base64_encode_images?
+    @base64_encode_images
+  end
+
+  def base64_encoded_uri(source)
+    asset = Rails.application.assets[source]
+
+    if asset.content_type == 'image/svg+xml'
```

Comment:
This is more about base64 reliably taking 8/6ths of space, while URL-encoding a SVG tends to preserve most bytes, but for about a fifth, it makes them 3 times larger.

But really, my concern here is mostly about having a hand-rolled solution to encode URLs in 2 different ways. If we used something ..established, I don't care what the resulting format is.